client/src/locale/ru-RU.locale.ts

```diff
@@ -0,0 +1,134 @@
+import Locale from './locale'
+
+export default <Locale>{
+    home: {
+        title: 'Pacstall',
+        subtitle: 'AUR для Ubuntu',
+        cards: {
+            whyDifferent: {
+                title: 'Чем это отличается от любого другого пакетного менеджера?',
+                description:
+                    'Pacstall использует стабильную базу Ubuntu, но позволяет использовать ' +
+                    "bleeding-edge программы почти что без компромиссов, так что вам не " +
+                    'нужно беспокоиться о патчах безопасности или новом функционале.',
+            },
+            howItWorks: {
+                title: 'Тогда как это работает?',
+                description:
+                    ' Pacstall takes in files known as <0>pacscripts</0> (similar' +
+                    ' to PKGBUILDs) that contain the necessary contents to build packages,' +
+                    ' and builds them into executables on your system.',
```

Comment:
Why can't these be translated?

Reply:
my bad, forgot about those, will fix in a bit!

Reply:
fixed now!